### PR TITLE
deal with languages better

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn ready for review

File: unicoremc/constants.py [hunk 1]
```diff
@@ -2,25 +2,444 @@
...
+    "srd": "Sardinian",
+    "scc": "Serbian",
+    "srr": "Serer",
+    "ssa": "Nilo-Saharan",
+    "ssw": "Swati",
+    "suk": "Sukuma",
+    "sun": "Sundanese",
+    "sus": "Susu",
+    "sux": "Sumerian",
+    "swh": "Swahili",  # using iso639-3 for swahili
```
Comment (smn) [inline unicoremc/constants.py:378]: I'm -1 on this, it's going to bite us badly. Let's fix this properly.

**Comment by smn** [comment]
@nathanbegbie & @codiebeulaine just pinging you so you can watch along.

File: unicoremc/manager.py [hunk 2]
```diff
@@ -187,7 +187,8 @@ def write_frontend_settings(
 
         languages = []
         for lang in available_languages:
-            languages.append("('%s', '%s')" % (lang.get_code(), str(lang)))
+            languages.append(
+                "('%s', '%s')" % (lang.get_code(), lang.get_display_name()))
```
Comment (smn) [inline unicoremc/manager.py]: [repr](https://docs.python.org/2/library/functions.html#repr) does that for you:

``` python
>>> repr(('swa', 'Swahili'))
"('swa', 'Swahili')"
```

**Comment by smn** [comment]
You can ignore the `repr` bikeshed if you'd prefer but having 1 language be different from the ISO standard we've committed to seems like a thing that'll bite us badly. Now is probably the least worst time to fix it.

File: unicoremc/constants.py (hunk 1 above)
Comment (miltontony) [inline unicoremc/constants.py:378]: @smn we'll have to make changes to existing content on all the apps that use swahili. We'll also need to inform FB about the change in code for their urls. And probably add a fix to `unicore-cms` to automatically change `swh` to `swa` when setting the locale

Comment (smn) [inline unicoremc/constants.py:378]: Yes, we can do a redirect for `swh_TZ` / `swh_KE` to `swa_*` and that'll be it. We'd need to notify FB but not require immediate action.

Comment (miltontony) [inline unicoremc/constants.py:378]: @smn cool. I'll create a new PR for this

Comment (miltontony) [inline unicoremc/constants.py:378]: @smn issue #41 created and issue [universalcore/unicore-cms#60](https://github.com/universalcore/unicore-cms/issues/60)

**Comment by miltontony** [comment]
@smn ready again. Using `repr` now and issues created to fix swahili

**Comment by smn** [comment]
:+1: thanks!

**Pull request merged**